src/App.js

```diff
@@ -1,19 +1,42 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
 
 const App = () => {
+  const [messages, setMessageData] = useState(chatMessages);
+  const [likesCount, setLikesCount] = useState(0);
```

Comment:
The interesting part here is that we actually don't necessarily need to make a state variable for the likes. We can simply calculate this with a function when the page renders. I will elaborate further in a later comment!